### ISSUE TITLE
Chore: add 2020 note data

### ISSUE THREAD

**Issue by seanpm2001** [seanpm2001/Git-Templates#2]
***

# Chore: add 2020 note data

This is a personal chore for myself, I won't need public help.

I plan on making the `/Git-Templates/GitHub/Notes/` directory more of an archive by including 2020 data in its full extent, obsoleting the [Seanpm2001/Notes/](https://github.com/seanpm2001/seanpm2001/Notes/) directory.

Some side info

I previously commented on Ruffle-RS to not consider things a chore. I was too supportive back then and ignored certain logic. Not all chores are bad, and I don't feel like I didn't do any harm to the ruffle community, looking back, the comment wasn't bad. Here is a link to the thread: [4](https://github.com/ruffle-rs/ruffle/pull/3194/)

In other words, I didn't fully understand the usage of the term.

***
